product-service/handlers/getAllProducts.ts

```diff
@@ -0,0 +1,29 @@
+import { APIGatewayProxyHandler } from 'aws-lambda';
+import 'source-map-support/register';
+import productList from '../productList.json';
+
+export const getAllProducts: APIGatewayProxyHandler = async (
+	event,
+	_context
+) => {
+	function getAsync() {
+		return new Promise((resolve) => setTimeout(() => resolve(productList), 0));
+	}
+
```

Comment:
One option that I can advise you is to have a call for retrieving productList from github - https://raw.githubusercontent.com/SychevAndrey/nodejs-aws-be/eeb932abdf5295aabc2e669f0b952c2a3c81dd81/product-service/productList.json